src/Install/Console/InstallCommand.php

```diff
@@ -133,7 +133,8 @@ protected function install()
                     'host' => 'required',
                     'database' => 'required|string',
                     'username' => 'required|string',
-                    'prefix' => 'alpha_dash|max:10'
+                    'prefix' => 'alpha_dash|max:10',
+                    'port'   => 'required|integer|min:1|max:65535',
```

Comment:
The port shouldn't be required. Just use the default (after trying to extract one from the hostname) if none was provided.